src/openrct2/ride/Vehicle.cpp

```diff
@@ -7739,15 +7736,15 @@ static bool vehicle_update_motion_collision_detection(
             if (y_diff > 0x7FFF)
```

Comment:
The tests are failing, I'm guessing because in the block above (lines 7714~7735) there are `x`s, `y`s and `z`s that should have been converted to  `curX`, `curY` and `curZ`

Reply:
Yes, thanks for pointing this out!